src/hub_api.rs

```diff
@@ -1,6 +1,6 @@
 use itertools::Itertools;
 
-const DEV_SITE_BASE: &'static str = "https://developer.fermyon.com";
+const DEV_SITE_BASE: &'static str = "http://127.0.0.1:3000";
```

Comment:
We probably don't want to commit in this form, but I'm not sure of the best way to switch between "the developer site" and "my local copy of the developer site" - thoughts?

Reply:
We still need to tackle this - the URL that gets built into the plugin _must_ be the real Hub.